### PR TITLE
Ngo edit page to allow contact edits 213

### PR DESCRIPTION
**Pull request by Thrillberg**

### CODE REVIEW

File: app/models/contact.rb [hunk 1]
```diff
@@ -3,11 +3,11 @@ class Contact < ApplicationRecord
...
   belongs_to :ngo
 
-  validates :first_name, length: { in: 1..50 }
-  validates :last_name, length: { in: 1..50 }
-  validates :email, presence: true
-  validates :phone, presence: true
-  validates :street, presence: true
-  validates :zip, presence: true
-  validates :city, presence: true
+  validates :first_name, length: { in: 1..50 }, on: :create
```
Comment (klappradla) [inline app/models/contact.rb]: Hey @Thrillberg, these validations are actually mandatory - we needs them on update as well. Could you please just revert this one file?

Comment (Thrillberg) [inline app/models/contact.rb]: :heavy_check_mark:

**Comment by klappradla** [comment]
Cool thanx @Thrillberg !
I'm at a conference at the moment - don't really have time to check out the code now, but will as soon as I get back 🚀

File: app/views/ngos/registrations/edit.html.erb [hunk 2]
```diff
@@ -1,34 +1,29 @@
...
-
-  <%= f.input :password, autocomplete: "off",
-                         hint: "(" + t("devise.registrations.edit.leave_blank_if_you_don_t_want_to_change_it") + ")" %>
-
-  <%= f.input :password_confirmation, autocomplete: "off" %>
-
-  <%= f.input :current_password, autocomplete: "off",
-                                  hint: "(" + t("devise.registrations.edit.we_need_your_current_password_to_confirm_your_changes") + ")" %>
-
-  <%= f.submit t("devise.registrations.edit.update"), class: "btn btn-primary" %>
```
Comment (klappradla) [inline app/views/ngos/registrations/edit.html.erb]: Hey @Thrillberg I just checked out your changes - nicely done, thanx a lot 👍 
However, I think you accidentally removed this line. So currently the edit form does not have a submit button 🙈  - could you please add it back in and check that it's working? Thanx a lot!

File: app/views/ngos/registrations/edit.html.erb [hunk 3]
```diff
@@ -1,34 +1,29 @@
...
+      </div>
+      <div class="col-sm-6">
+        <%= render 'ngo_fields', f: f %>
+        <%= f.input :locale, as: :select, collection: Ngo.locales.keys.to_a %>
+        <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
+          <div>Erwarten Bestätigung für: <%= resource.unconfirmed_email %></div>
+        <% end %>
+        <%= f.input :current_password, autocomplete: "off",
+                                       hint: "(" + t("devise.registrations.edit.we_need_your_current_password_to_confirm_your_changes") + ")" %>
+        <h3><%= t("devise.registrations.edit.cancel_my_account") %> </h3>
```
Comment (klappradla) [inline app/views/ngos/registrations/edit.html.erb]: Having this huge headline above the *meant to be* more subtle `cancel account` button is a bit overkill in my opinion. Having the text on the button is enough I think. Or do you see a special benefit in having this? I'd prefer removing the `h3` line.

**Comment by klappradla** [comment]
Hey @Thrillberg I checked out your PR and had a look at the code and web app - sorry for taking so long, as I said, I was at a conference and pretty busy.

Huge 💚  for your work, really appreciate it!

File: app/views/ngos/registrations/edit.html.erb (hunk 2 above)
Comment (Thrillberg) [inline app/views/ngos/registrations/edit.html.erb]: Oh, how embarrassing! Thanks for noticing this. I've added it back and checked that it's working.

**Comment by klappradla** [comment]
Looks good @Thrillberg thanx for you patience ✌️

**Pull request merged**